docs/source/models.rst

```diff
@@ -1600,6 +1620,37 @@ For more info, see:
     mlflow models predict --help
     mlflow models build-docker --help
 
+Environment Management Tools
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MLflow currently supports the following environment management tools to restore model environments:
+
+local
+    Use the local environment. No extra tools are required.
+conda
+    Create environments using conda. Conda must be installed for this mode of environment reconstruction.
+
+    - `conda installation instructions <https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html>`_
+virtualenv
+    .. note::
```

Comment:
Nit: Can we move the `note` to the bottom of the `virtualenv` section?

Reply:
Sure!